### ISSUE TITLE
Support for building inside Bash for Windows

### ISSUE THREAD

**Issue by sigboe** [kiibohd/controller#150]
Windows 10 update dubbed anniversary edition (horrible name) now supports installing Bash for Windows, which is developed in cooperation with Canonical, and works in effect similar to a chroot in Linux. It can be installed through `Control Panel -> Programs and Features -> Turn Windows features on or off` and practically installs Ubuntu in this chroot like environment. 

It can be used for many things that cygwin can be used for, but I don't know the exact compatiblity. New applications can be installed from debian repositories (presumably they must be built for Windows? idk) jusing commands like `apt installl` and `apt-get install`.

Is compiling the firmware supported using Bash for Windows? If not, maybe compatibility issues should be looked into.

**Comment by haata** [comment]
Yes, it's supported. I've tried it a few times. Should work exactly like
Ubuntu.

There are a couple caveats.
1. The load script doesn't work (haven't gotten lol libusb working).
2. Not sure screen works for the virtual CLI
